tron/config/config_parse.py

```diff
@@ -67,9 +67,13 @@ def validator(value, config_context):
         )
 
         try:
-            value % context
+            render_command = value % context
+            # This checks wrong format like "$(A) $(A)s"
+            if render_command != (render_command % context):
+                raise TypeError("type is not specified")
+
             return value
-        except (KeyError, ValueError) as e:
+        except (KeyError, ValueError, TypeError) as e:
```

Comment:
I agree we should also catch TypeError, but maybe in a separate case? The error_msg doesn't really make sense when it's a type error.

Reply:
Yeah, good point.